src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, ComponentProps } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import * as DropdownSelect from '@radix-ui/react-select';
+import ChevronDownIcon from '../../assets/chevronDown.svg?react';
+import ChevronUpIcon from '../../assets/chevronUp.svg?react';
+
+export interface DropdownProps
+  extends Omit<DropdownSelect.SelectProps, 'defaultValue' | 'dir'>,
+    Omit<ComponentProps<'div'>, 'ref'> {
+  /**
+   * All options to be listed in the dropdown
+   */
+  options: {
+    label: string;
+    value: string;
```

Comment:
could we export this as it's own typescript type for consumers to use?  `export interface DropdownItem { label: string; value: string } `